webpack/dev.config.js

```diff
@@ -8,7 +8,7 @@ const customPath = path.join(__dirname, './customPublicPath');
 const hotScript = 'webpack-hot-middleware/client?path=/__webpack_hmr&dynamicPublicPath=true';
 
 const baseDevConfig = () => ({
-  devtool: 'eval-cheap-module-source-map',
+  devtool: 'eval-source-map',
```

Comment:
@jeffreydunaway - The original reference in the config is a typo taken from the boilerplate, it should actually be `cheap-module-eval-source-map` as per the docs.

For some strange reason at the time, it wasn't firing off the debugger when I tried the correct value so settled for `eval-source-map`. 

Just tried with the actual value and it seems to be working fine now, will probably switch it over at the some point. There isn't any noticeable performance regression running the webpack server for me, how about you?